app/helpers/api/ordapi/postOrderItemBulk.js

```diff
@@ -40,20 +39,23 @@ export const postOrderItemBulk = async ({
   recipients,
   formData,
 }) => {
-  const endpoint = getPostOrderItemEndpoint(orderId);
+
+  const endpoint = `${orderApiUrl}/api/v1/orders/${orderId}/order-items/${itemId}`;
```

Comment:
It would be better if `itemId` was `catalogueItemId` but we can create a tech debt task for that if necessary as it would be good to get this fix merged.